hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/clustering/ClusteringOperator.java

```diff
@@ -233,6 +233,7 @@ private void doClustering(String instantTime, ClusteringPlanEvent event) throws
 
     List<WriteStatus> writeStatuses = this.writerHelper.getWriteStatuses(this.taskID);
     collector.collect(new ClusteringCommitEvent(instantTime, writeStatuses, this.taskID));
+    this.writerHelper = null;
   }
 
```

Comment:
Is there any possibility we write some tests, say when there are multiple clustering plan to execute.